api/src/main/java/com/datatorrent/api/Context.java

```diff
@@ -328,6 +328,8 @@
      */
     int getWindowsFromCheckpoint();
 
+    String getOperatorName();
```

Comment:
The same comment applies to other introduced `operatorName` fields where `id` means operator Id, while `operatorName` is used for an operator name.